.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+ADO card: ☑️ [AB#XXXX](https://dev.azure.com/S72/SHIFT72/_boards/board/t/Delivery%20team/Stories/?workitem=XXXX)
+
+Elab notes/AC: 📃 [Google Docs](https://docs.google.com)
+
+Designs: 
+- 📱 [Mobile](http://www.zeplin.io/????)
+- 💊 [Tablet](http://www.zeplin.io/????)
+- 🖥 [Desktop](http://www.zeplin.io/????)
+
+## Description of work
+This is a description of the feature or bug, including any detail not in the ADO card that is required to understand the work.
+
+## Edge cases/Caveats/Known issues
+- Edge case 1
+- Caveat 2
+- Known issue 3
+
+## Config settings/Toggles required for the feature to work
+- Config setting 1
+
+## Related PRs 
+
+### Any PRs which this PR depends on
+
+### Any PRs dependent on this one
+
+### Affected Clients
+ - All clients who use Kibble
+ - Client 1
+ - Client 2
+
+## Tests
+📈 Coverage increased from XX% to YY%
```

Comment:
😬

Reply:
I'll remove that for now.  I think we should formalise the `scripts` at some point though and add tests.